README.md

```diff
@@ -4,6 +4,7 @@
 
 [![Lifecycle: stable](https://img.shields.io/badge/lifecycle-stable-brightgreen.svg)](https://lifecycle.r-lib.org/articles/stages.html#stable)
 [![R-CMD-check](https://github.com/RMI-PACTA/pacta.data.scraping/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/RMI-PACTA/pacta.data.scraping/actions/workflows/R-CMD-check.yaml)
+[![codecov](https://codecov.io/gh/RMI-PACTA/pacta.data.scraping/graph/badge.svg?token=PL3S8FGNDJ)](https://codecov.io/gh/RMI-PACTA/pacta.data.scraping)
```

Comment:
Is it sketchy to have the token committed directly to the README?

Reply:
Seems probably like it is...

Reply:
It's not the same as the secret "upload token"... I believe that token is what it uses to determine what branch you want to show coverage for? It's copy-pasted straight from their settings interface that tells you what to put in your README... e.g. https://app.codecov.io/gh/RMI-PACTA/pacta.data.scraping/settings/badge

Reply:
hmmmm.... but it does seem like maybe you don't need a token for the default branch?
https://docs.codecov.com/docs/status-badges

Reply:
I'm gonna remove it and see what happens